examples/misc/logos2.py

```diff
@@ -8,6 +8,104 @@
 Thanks to Tony Yu <tsyu80@gmail.com> for the logo design
 """
 
+import numpy as np
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import matplotlib.cm as cm
+from matplotlib.patches import Circle, Rectangle
+
+
+def make_logo(pixels, lw_bars, lw_grid, lw_border, rgrid):
+    mpl_blue = '#11557c'
+    dpi = 100
+
+    with plt.rc_context({'axes.edgecolor': mpl_blue,
+                         'axes.linewidth': lw_border}):
+        fig = plt.figure(figsize=(pixels / dpi, pixels / dpi), dpi=dpi)
+        fig.patch.set_alpha(0)
+
+        ax = fig.add_axes((0.03, 0.03, .94, .94), projection='polar')
+        ax.set_axisbelow(True)
+
+        N = 7
+        arc = 2. * np.pi
+        theta = np.arange(0.0, arc, arc / N)
+        radii = np.array([2, 6, 8, 7, 4, 5, 8])
+        width = np.pi / 4 * np.array([0.4, 0.4, 0.6, 0.8, 0.2, 0.5, 0.3])
+        bars = ax.bar(theta, radii, width=width, bottom=0.0, align='edge',
+                      edgecolor='0.1', lw=lw_bars, )
+        for r, bar in zip(radii, bars):
+            color = *cm.jet(r / 10.)[:3], 0.6  # color from jet with alpha=0.6
+            bar.set_facecolor(color)
+
+        ax.tick_params(labelbottom=False, labeltop=False,
+                       labelleft=False, labelright=False)
+
+        ax.grid(lw=lw_grid, ls=':', color='0.7')
+        ax.set_rmax(9)
+        ax.set_yticks(rgrid)
+
+        # the actual visible background - extends a bit beyond the axis
+        ax.add_patch(Rectangle((0, 0), arc, 9.58,
+                               facecolor='white', zorder=0,
+                               clip_on=False, in_layout=False))
+        return fig, ax
+
+
+params_large = {
+    'pixels': 300,
+    'lw_bars': 2,
+    'lw_grid': 0.8,
+    'lw_border': 3,
+    'rgrid': [1, 3, 5, 7],
+}
+fig, ax = make_logo(**params_large)
+plt.savefig('logo_large.png')
+
+##############################################################################
+
+params_medium = {
+    'pixels': 64,
+    'lw_bars': 0.5,
+    'lw_grid': 0.5,
+    'lw_border': 1,
+    'rgrid': [3, 6],
+}
+
+make_logo(**params_medium)
+plt.savefig('logo_medium.png')
+
+##############################################################################
+
+params_small = {
+    'pixels': 32,
+    'lw_bars': 0.3,
+    'lw_grid': 0.3,
+    'lw_border': 0.3,
+    'rgrid': [5],
+}
+
+make_logo(**params_small)
+plt.savefig('logo_small.png')
+
+##############################################################################
+
+params_xsmall = {
+    'pixels': 16,
+    'lw_bars': 0.1,
+    'lw_grid': 0.3,
+    'lw_border': 0.3,
+    'rgrid': [],
+}
+
+make_logo(**params_xsmall)
+plt.savefig('logo_xsmall.png')
+
+##############################################################################
+#
```

Comment:
Maybe we can just remove the old stuff? People can always look at our old documentation to find it.

Reply:
That's the plan. I'll clean up the PR once I've found time to get the text in.

Reply:
:+1: